timesketch/lib/analyzers/interface.py

```diff
@@ -351,6 +353,26 @@ def add_view(self, view_name, analyzer_name, query_string=None,
         db_session.commit()
         return view
 
+    def add_story(self, title):
+        """Add a story to the Sketch.
+
+        Args:
+            title: The name of the view.
+
+        Raises:
+            ValueError: If both query_string an query_dsl are missing.
+
+        Returns:
+            An instance of a Story object.
+        """
+
+        story = SQLStory.get_or_create(
+            title=title, content=json.dumps([]), sketch=self.sql_sketch,
```

Comment:
I don't see a reason to call `json.dumps([])` here, that is a constant... calling `json.dumps([])` will produce the string: `'[]'`, and therefore just have `content='[]', ` should suffice

Reply:
good catch, fixed.